be/src/exec/pipeline/query_context.h

```diff
@@ -93,6 +93,7 @@ class QueryContext : public std::enable_shared_from_this<QueryContext> {
     void set_enable_pipeline_level_shuffle(bool flag) { _enable_pipeline_level_shuffle = flag; }
     bool enable_pipeline_level_shuffle() { return _enable_pipeline_level_shuffle; }
     void set_enable_profile() { _enable_profile = true; }
+    bool get_enable_profile_flag() { return _enable_profile; }
     bool enable_profile() {
```

Comment:
enable_profile is enough, no need to to define a get_enable_profile_flag. can we considerate that load task resemble big query, so we should throttle the times of status reports carrying load profiles, otherwise, FE would be overwhelmed by too much load profiles. especially, when users create large number of MV(s).

Reply:
1. The result of `enable_profile()` is dynamic because it will compare `_big_query_profile_threshold_ns` with query run time. Here just want to get the original configs
2. Load channel profile report is also controlled by `big_query_profile_threshold_ns` and `runtime_profile_report_interval`. We can throttle the reports by adjust these variables